lagrangebench/train/strats.py

```diff
@@ -38,17 +38,18 @@ def add_gns_noise(
         shift_fn: Shift function.
     """
     isl = input_seq_length
-    # random-walk noise in the velocity applied to the position sequence
+    # random-walk noise in the velocity applied to the position sequence w/o pushforward
```

Comment:
This can be misleading (e.g. can look like it's applied only without pushforward), what about 
```suggestion
    # random-walk noise in the velocity applied to the base input position sequence
```

Reply:
```suggestion
    # random-walk noise in the velocity applied to the first input_seq_length positions
```